### PR TITLE
[Frontend] Posts/Announcements from Modals appear on screen + smaller fixes

### PR DESCRIPTION
**Pull request by carmen-lee**
# [Frontend] | Made modals dynamic
> Team member: @carmen-lee 
> Use Case ID:

Fixes #[Issue]

## Expected Behavior
After clicking on "CREATE ANNOUNCEMENT" or "CREATE POST" on the Home/Feed page, the announcement and post will appear on screen. Posts will be deleted when the app restarts

- Added keys prop to event details page
- added profile pictures to orgs on the Home tab
- changed Home and Feed files to use functional components instead of class components (for my convenience)
- made the styles of the cards on the Home tab and Feed tab the same
- removed the "Following" and "All" toggle on Home tab
- edited constants file for the Home tab
- changed the icon for Explore tab
- removed badge number from tab icons

Simulator Screen Shot - iPhone 14 - 2023-04-19 at 21 37 26
Simulator Screen Shot - iPhone 14 - 2023-04-19 at 21 37 47

### CODE REVIEW

**Review by cerasamson — APPROVED** [review]
Works really well! nice job

File: frontend/VolunteerOne/components/EventDetails/index.js [hunk 1]
```diff
@@ -20,19 +20,19 @@ const EventDetails = ({ eventID }) => {
   if (eventID in events) {
     let eventDetails = events[eventID];
     //retrieves and formats the body of the event post
-    let bodyContent = eventDetails["eventBody"].map(function (data) {
+    let bodyContent = eventDetails["eventBody"].map((data, i) => {
```
Comment (Mtessier809) [inline frontend/VolunteerOne/components/EventDetails/index.js:23]: I made this fix in my last pr.

**Review by Mtessier809 — APPROVED** [review]
Looks great to me

**Review by mattacy — APPROVED** [review]
Great work!!

**Pull request merged**